### PR TITLE
BOM for the current set of packages

### PR DESCRIPTION
**Pull request by pallavit**

### CODE REVIEW

File: sdk/boms/azure-sdk-bom/pom.xml [hunk 1]
```diff
@@ -28,213 +27,205 @@
...
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-communication-identity</artifactId>
+        <artifactId>azure-communication-chat</artifactId>
         <version>1.1.0</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-communication-phonenumbers</artifactId>
```
Comment (pallavit) [inline sdk/boms/azure-sdk-bom/pom.xml:71]: We are working on adding phonenumbers, sms and identity.

**Review by alzimmermsft — APPROVED** [review]

File: sdk/boms/azure-sdk-bom/pom.xml (hunk 1 above)
Comment (pallavit) [reply sdk/boms/azure-sdk-bom/pom.xml:71]: All communication libraries have been released.

File: sdk/boms/azure-sdk-bom/pom.xml [hunk 2]
```diff
@@ -28,213 +27,205 @@
...
+        <version>1.1.6</version>
       </dependency>
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core-serializer-json-jackson</artifactId>
-        <version>1.2.4</version>
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-core-tracing-opentelemetry</artifactId>
```
Comment (pallavit) [inline sdk/boms/azure-sdk-bom/pom.xml:116]: Removing opentelemetry from the BOM.

**Comment by pallavit** [comment]
/cc: @alzimmermsft , @lmolkova , @srnagar , @JonathanGiles please take a look.

File: sdk/boms/azure-sdk-bom/pom.xml [hunk 3]
```diff
@@ -28,213 +27,205 @@
...
       <dependency>
         <groupId>com.azure</groupId>
-        <artifactId>azure-communication-phonenumbers</artifactId>
-        <version>1.0.3</version>
-      </dependency>
-      <dependency>
-        <groupId>com.azure</groupId>
-        <artifactId>azure-communication-sms</artifactId>
-        <version>1.0.3</version>
+        <artifactId>azure-communication-common</artifactId>
```
Comment (alzimmermsft) [inline sdk/boms/azure-sdk-bom/pom.xml:62]: Does Phone Numbers and SMS need to be added back?

**Review by alzimmermsft — APPROVED** [review]

**Comment by pallavit** [comment]
Thank you for the review.

**Pull request merged**